docs/fabric-native-components-ios.md

```diff
@@ -212,10 +212,10 @@ Update the `AppDelegate.mm` to make your application aware of our custom WebView
 #import "AppDelegate.h"
 
 #import <React/RCTBundleURLProvider.h>
+// highlight-start
 #import <React/RCTBridge+Private.h>
-// highlight-next-line
 #import "RCTWebView.h"
-
+// highlight-end
```

Comment:
I had to add the `RCTWebView` header on my machine, so I moved the highlight to encapsulate it.